pages/_app.tsx

```diff
@@ -38,8 +38,7 @@ export default function App(props: AppProps) {
             }
             styles={(theme) => ({
               main: {
-                backgroundColor:
-                  theme.colorScheme === "dark" ? theme.colors.dark[8] : theme.colors.gray[0],
+                backgroundColor: theme.colors.cyan[0],
```

Comment:
This is opinionated, but I think the Cyan buttons on Cyan background is a little Cyan overload for me. Can we play around with the colors here a bit, maybe find like a light gray or something that doesn't pop too much?

![image](https://user-images.githubusercontent.com/16297930/176245888-297502cb-7aa8-4287-b90e-86cbb12cc044.png)